includes/html.inc

```diff
@@ -32,15 +32,43 @@ function gesso_preprocess_html(array &$variables) {
  */
 function gesso_page_attachments_alter(array &$attachments) {
   $include_back_to_top = theme_get_setting('include_back_to_top') ?? TRUE;
-  if ($include_back_to_top) {
-    $attachments['#attached']['drupalSettings']['gesso']['backToTopThreshold']
-      = theme_get_setting('threshold', 'gesso');
-    $attachments['#attached']['drupalSettings']['gesso']['backToTopSmoothScroll']
-      = theme_get_setting('smooth_scroll', 'gesso');
+  $add_external_link_icons = theme_get_setting('add_external_link_icons') ?? FALSE;
+
+  if ($include_back_to_top || $add_external_link_icons) {
```

Comment:
I don't think you need the outer if statement here, since the two don't have any shared code.

Reply:
I believe they both share this:
```    /** @var \Drupal\Core\Render\Renderer $renderer */
    $renderer = \Drupal::service('renderer');
    $themeConfig = \Drupal::config('gesso.settings');
    $renderer->addCacheableDependency($attachments, $themeConfig);
```